packages/cli/src/commands/site/version/list/versionListCommand.mts

```diff
@@ -55,7 +55,10 @@ export default async function versionListCommand(
   }
 
   consola.success("Found versions:");
-  for (const version of versions) {
+
+  const semver = await import("semver");
+  const sortedVersions = semver.rsort(versions.filter(v => semver.valid(v)));
```

Comment:
We can probably assume the versions are enforced semver given the timelines?

Reply:
I think its better to leave it and I could clarify with a comment. I personally have few broken apps that have LOOSE versions stored

Reply:
Yeah I guess the point is that after a certain point in time in the future we're not going to have the control to filter this and it will be on the service layer to decide how to handle this instead